.gitignore

```diff
@@ -1,8 +1,6 @@
 node_modules
 yarn-*.log
 lerna-*.log
-/site/.cache/
-/site/public/
 /storybook/public/
```

Comment:
We should also call out in WOL just to make sure no one merges in these files after we merge this in 😉

Reply:
True! Good call 📞